.idea/inspectionProfiles/Project_Default.xml

```diff
@@ -0,0 +1,81 @@
+<component name="InspectionProjectProfileManager">
```

Comment:
The opening and closing tag for the component name should be on separate lines for better readability.